assets/js/modules/sign-in-with-google/components/settings/SettingsView.js

```diff
@@ -173,7 +147,7 @@ export default function SettingsView() {
 				<div className="googlesitekit-settings-module__meta-item">
 					<h5 className="googlesitekit-settings-module__meta-item-type">
 						{ __(
-							'Users can create new accounts',
+							'Anyone can register WP setting',
```

Comment:
Not sure why this changed but "Users can create new accounts" is definitely the clearer/better string to use, and one we decided on here: #9741.